test/system/libs/helpers.bash

```diff
@@ -27,6 +30,20 @@ function cleanup_containers() {
 }
 
 
+function setup_containers_store() {
+  mkdir -p ${TEMP_STORAGE_DIR}
+  # Setup a storage config file for PODMAN
+  echo -e "[storage]\n  driver = \"overlay\"\n  rootless_storage_path = \"${ROOTLESS_PODMAN_STORE_DIR}\"\n" > ${PODMAN_STORE_CONFIG_FILE}
+  export CONTAINERS_STORAGE_CONF=${PODMAN_STORE_CONFIG_FILE}
+}
+
+
+function _clean_containers_store() {
+  cleanup_all
```

Comment:
The environment is not set up properly so this will blast the host machine of all Podman state.